docs/getting_started/Troubleshooting.md

```diff
@@ -0,0 +1,33 @@
+# Troubleshooting
+
+TODO add motivational/introduction text.
```

Comment:
Any ideas on how we can write this?

Since this is the getting started section, maybe we should move this to another section?